src/main/java/com/bobi89/medicalclinic/service/patient_service/PatientServiceImpl.java

```diff
@@ -85,6 +91,49 @@ public ChangePasswordCommand editPatientPassword(String email, ChangePasswordCom
         return pass;
     }
 
+    @Transactional
+    @Override
+    public PatientDTO addAppointmentToPatient(LocalDateTime dateTime, int durationMinutes,
```

Comment:
do wywalenia chyba ta metoda -> mielismy uzyc query a nie findAll() <- uzywanie findAll najczesciej jest błędem. W przypadku tabel, które mogą mieć bardzo dużo rekordów <- jak niewątpliwie tabela Appointments może to doprowadzić, że będziemy próbowali pobrać tak dużo rekordów do pamięci, że wywali nam apkę